Werehouse/Werehouse.py

```diff
@@ -0,0 +1,50 @@
+helmet = 0
+rifle = 0
+medkit = 0
+
+print('Hello in our werehouse')
+while True:
+    print("We have: " + str(helmet) + " helmets, " + str(rifle) + " rifles, and " + str(medkit) + " medkits"  )
+    Temp = input("Type 1 to add or 2 for out item: ")
+    if Temp == "1":
+
+        Temp1 = input("Type item to add : ")
+        if Temp1 == "helmet":
+            Temp2 = input("how many helmets would You like to add :")
+            helmet += int(Temp2)
+        elif Temp1 == "rifle":
+            Temp2 = input("how many rifles would You like to add :")
+            rifle += int(Temp2)
+        elif Temp1 == "medkit":
+            Temp2 = input("how many medkits would You like to add :")
+            medkit += int(Temp2)
+        else:
+            print("You give wrong imput, use: helmet, rifle or medkit, try again")
+
+    elif Temp == "2":
+        Temp1 = input("Type item to out : ")
+        if Temp1 == "helmet":
+            Temp2 = input("how many helmets would You like to out :")
+            if helmet <= 0:
+                print("there is no helmets!")

```

Comment:
I think `if` checking the state of the warehouse should be before asking the user for the input - if we have 0 items in the warehouse there is no point for asking the user how many should be withdrawn

Reply:
Yes, its displays in the beginning :)